src/js/components/FormField/doc.js

```diff
@@ -42,6 +42,9 @@ export const doc = FormField => {
     pad: PropTypes.bool.description(
       'Whether to add padding to align with the padding of TextInput.',
     ),
+    placeholder: PropTypes.node.description(
```

Comment:
`placeholder` isn't a prop of FormField and, should be removed from FormField associated props.

Reply:
okay so we just want to accept it for TS maybe we can discuss